.github/workflows/release.yml

```diff
@@ -807,3 +807,4 @@ name: release
   push:
     tags:
       - v*.*.*
+  workflow_dispatch: {}
```

Comment:
Wouldn't this actually publish docs now and release notes?
`IS_PRERELEASE: ${{ contains(github.ref_name,'-') }}` I'm not sure what this resolves to for an manual trigger.